pipeline/plot/unit_characteristic_plot.py

```diff
@@ -129,20 +129,33 @@ def plot_unit_selectivity(probe_insert_key, axs=None):
 
 
 def plot_unit_bilateral_photostim_effect(probe_insert_key, axs=None):
-    trial_restrictor = {'task': 'audio delay', 'task_protocol': 1, 'early_lick': 'no early'}  # TODO: confirm this restrictor
-    both_alm_stim_res = experiment.Photostim * experiment.BrainLocation & 'brain_location_name = "both_alm"'
 
-    no_stim_trials = experiment.BehaviorTrial - experiment.PhotostimTrial & trial_restrictor
-    bi_stim_trials = experiment.BehaviorTrial * experiment.PhotostimTrial & trial_restrictor & both_alm_stim_res
+    no_stim_cond = (psth.TrialCondition
+                    & {'trial_condition_desc':
+                       'all_noearlylick_both_alm_nostim'}).fetch1('KEY')
+
+    bi_stim_cond = (psth.TrialCondition
+                    & {'trial_condition_desc':
+                       'all_noearlylick_both_alm_stim'}).fetch1('KEY')
 
     units = ephys.Unit & 'unit_quality = "good"'
```

Comment:
Missing restriction by `probe_insert_key`

Reply:
this is the condition key; not actual 'expansion' into trials

relatedly TrialCondition yields a query and not a fetch so if it were needed, one would:

`(psth.TrialCondition.get_trials(...) & probe_insert_key).fetch()`

perhaps TrialCondition functions should be renamed... hmm.

Reply:
side note the other, possibly confusing usage when casually viewed here, usage of trial condition where restriction is applied occurs in UnitPsth, where it is actually an upstream filter on the condition:

```        # expand TrialCondition to trials,
        trials = TrialCondition.get_trials(
            (TrialCondition & key).fetch1('trial_condition_desc'))
```